### ISSUE TITLE
Add congressional district to property information

### ISSUE THREAD

**Issue by dodgees** [opennebraska/landlord#89]
Add a way to display which congressional district where a property is located.

**Comment by dodgees** [comment]
@payne Would the work you did for Danni help here? I'm wondering if the code for that might serve as an example for this issue.

**Comment by payne** [comment]
Yes, I think this will be great.   I'll look into it this evening!  Thanks for tagging me.

**Comment by ed-snodgrass** [comment]
Hey @payne , Did you have some code that might help with this?

**Comment by payne** [comment]
Yes.  I'll be back next week.   My part time job has been busy lately.  See you on Tuesday 27-Apr-21.
